images/daos-server-image.pkr.hcl

```diff
@@ -70,18 +70,4 @@ build {
       "./scripts/install_daos.sh"
     ]
   }
-
-  provisioner "file" {
-    source = "./scripts/cert_gen"
-    destination = "/tmp/"
-  }
-
-  provisioner "shell" {
-    inline = [
-      "sudo mkdir -p /var/daos/",
-      "sudo mv /tmp/cert_gen /var/daos/",
-      "sudo chmod +x /var/daos/cert_gen/*.sh",
-      "sudo chown -R root:root /var/daos/cert_gen"
-    ]
-  }
```

Comment:
When installing DAOS with rpms the cert gen scripts are installed in /usr/lib64/daos/certgen/gen_certificates.sh

Therefore, it is no longer necessary to copy those scripts to /var/daos/cert_gen when building the image.